test/e2e/sdk/actorChargeLocal/test.mjs

```diff
@@ -17,7 +17,7 @@ const runActor = async (input = {}, options = {}) => {
     return await client.run(runId).get();
 };
 
-test('basic functionality', async () => {
+void test('basic functionality', async () => {
```

Comment:
should we maybe await the top-level ones instead? both looks weird to be honest, not a blocker, i am fine with the void keyword as well

Reply:
Maybe we could disable the rule for this specific case - https://github.com/nodejs/node/issues/51292#issuecomment-2241761056

Reply:
sounds good